The type or namespace name 'Thrift' does not exist in the namespace 'Jaeger.Senders'

**Describe the bug**
`Jaeger.Example.WebApi/Startup.cs(4,22): error CS0234: The type or namespace name 'Thrift' does not exist in the namespace 'Jaeger.Senders' (are you missing an assembly reference?)`

**To Reproduce**
Steps to reproduce the behavior:
1. clone repo locally
2. navigate to examples/Jaeger.Example.WebApi
3. run `dotnet run`
4. receive error `CSC : error CS8102: Public signing was specified and requires a public key, but no public key was specified. [aeger-client-csharp/examples/Jaeger.Example.WebApi/Jaeger.Example.WebApi.csproj]`
5. replace relative path package reference with Nuget package reference as described in the comment on lines 10-14 in Jaeger.Example.WebApi.csproj
6. receive error `Jaeger.Example.WebApi/Startup.cs(4,22): error CS0234: The type or namespace name 'Thrift' does not exist in the namespace 'Jaeger.Senders' (are you missing an assembly reference?)`

**Expected behavior**
The app runs.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Version (please complete the following information):**
 - OS: macOS
 - Jaeger version: [e.g. 1.8]
 - Deployment: [e.g. bare metal, Docker, Kubernetes, ...]

**What troubleshooting steps did you try?**
Try to follow https://www.jaegertracing.io/docs/latest/troubleshooting/ and describe how far you were able to progress and/or which steps did not work.

**Additional context**
Add any other context about the problem here.

--- a/examples/Jaeger.Example.WebApi/Jaeger.Example.WebApi.csproj
+++ b/examples/Jaeger.Example.WebApi/Jaeger.Example.WebApi.csproj
@@ -5,13 +5,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="OpenTracing.Contrib.NetCore" Version="0.6.2" />
+    <PackageReference Include="OpenTracing.Contrib.NetCore" Version="0.7.1" />
 
     <!-- NOTE:
         In case you're building this application standalone, use below PackageReference 
         with an appropriate version instead of relative-path ProjectReference 
         to Jaeger project -->
-    <!-- PackageReference Include="Jaeger" Version="0.2.1" /-->
+    <!-- PackageReference Include="Jaeger" Version="1.0.2" /-->
   </ItemGroup>
 
   <ItemGroup>

